### ISSUE TITLE
TypeError: Cannot read property 'extend' of undefined

### ISSUE THREAD

**Issue by pika89** [ursudio/webgl-heatmap-leaflet#23]
Hello, im trying to use your plugin on an ionic app thats using leaflet.js and openstreetmap. I am getting an error: 

TypeError: Cannot read property 'extend' of undefined

Could you help me with this?

**Comment by bozdoz** [comment]
Is the leaflet library loaded? Can you see a line number or a screenshot of the error?

**Comment by pika89** [comment]
Hi, thank you for answering, heres a screen shot of the error: 
![screenshot_1](https://user-images.githubusercontent.com/6146621/32043105-1a583016-ba3a-11e7-9006-b5f80ae8c71b.png)
Yes the leaflet library is loaded
![screenshot_2](https://user-images.githubusercontent.com/6146621/32043170-3d8f04f6-ba3a-11e7-82ee-0943215cb3c7.png)
![screenshot_3](https://user-images.githubusercontent.com/6146621/32043197-4e53eee6-ba3a-11e7-969f-911849c64235.png)
Heres another screenshot of the error: 
![screenshot_4](https://user-images.githubusercontent.com/6146621/32043240-76ea5e12-ba3a-11e7-926d-d66903eb7cbd.png)

**Comment by bozdoz** [comment]
What's your Leaflet version?

**Comment by pika89** [comment]
My leaflet version is 0.7.7

**Comment by pika89** [comment]
Ok it is no longer showing this error when i switched to 1.2.0 version but when i follow this example here: http://tombatossals.github.io/angular-leaflet-directive/examples/0000-viewer.html#/layers/webgl-heatmap-example , i don't see Heatmap in the overlay or on the map at all?

**Comment by bozdoz** [comment]
Do you see the heatmap in the link you sent? Is it just in your own local example that you can't see it?

**Comment by pika89** [comment]
In the example i can see the heatmap, in my local example i can't

**Comment by bozdoz** [comment]
Any new errors?

**Comment by pika89** [comment]
I'm getting these 2: 
DevTools failed to parse SourceMap: http://localhost:3000/bower_components/ui-leaflet-layers/dist/ui-leaflet-layers.js.map and angular.js:14199 [AngularJS - Leaflet] A layer must have a valid type defined

**Comment by bozdoz** [comment]
Sorry, it looks like this is now beyond my expertise.  I'm not familiar enough with AngularJS.

I can say it looks like it can't find a source map for a javascript file (which seems ridiculous), and is possibly related to a different plugin (ui-leaflet-layers)?

**Comment by pika89** [comment]
Yeah i've tried looking further into this but i haven't actually found anything related to my problem, i'm still not sure what's the problem. I've tried reinstalling all the plugins with bower and getting the updated versions but the issue still persists.

**Comment by idanielsteven** [comment]
I am also having the exact same issue with 0.7.7. Need to be able to use 0.7.7 for angular-leaflet-directive.
